lib/features/booking/add_inspection_booking_screen.dart

```diff
@@ -94,7 +94,14 @@ class _AddInspectionBookingBody extends State<AddInspectionBookingBody> {
                 ),
                 context: context,
                 builder: (_) {
-                  return const UpdatePhoneNoBottomSheetView();
+                  return BlocProvider.value(
+                      value: BlocProvider.of<MyProfileCubit>(context),
```

Comment:
Where is MyProfileCubit created? Can you be sure this cubit is always created at this time?

Reply:
I already changed cubit that will match with parent cubit. MyProfileCubit will not be used any more, instead, I'm using AddInspectionCubit